proposals/fluentd.adoc

```diff
@@ -0,0 +1,112 @@
+- [fluent/fluent-plugin-webhdfs](http://github.com/fluent/fluent-plugin-webhdfs)
+- [fluent/fluent-logger-java](http://github.com/fluent/fluent-logger-java)
+- [fluent/fluentd-benchmark](http://github.com/fluent/fluentd-benchmark)
+- [fluent/fluentd-forwarder](http://github.com/fluent/fluentd-forwarder)
+- [fluent/fluentd-ui](http://github.com/fluent/fluentd-ui)
+- [fluent/fluent-bit-demo](http://github.com/fluent/fluent-bit-demo)
+- [fluent/fluent-bit-dashboard](http://github.com/fluent/fluent-bit-dashboard)
+- [fluent/fluent-plugin-scribe](http://github.com/fluent/fluent-plugin-scribe)
+- [fluent/data-collection](http://github.com/fluent/data-collection)
+- [fluent/fluent-logger-d](http://github.com/fluent/fluent-logger-d)
+- [fluent/fluent-logger-ocaml](http://github.com/fluent/fluent-logger-ocaml)
+- [fluent/fluent-plugin-msgpack-rpc](http://github.com/fluent/fluent-plugin-msgpack-rpc)
+- [fluent/fluent-plugin-hoop](http://github.com/fluent/fluent-plugin-hoop)
+- [fluent/fluent-plugins](http://github.com/fluent/fluent-plugins)
+- [fluent/fluent.github.com](http://github.com/fluent/fluent.github.com)
+- [fluent/Nlog.Targets.Fluentd](http://github.com/fluent/Nlog.Targets.Fluentd)
+- [fluent/fluent-logger-scala](http://github.com/fluent/fluent-logger-scala)
+- [fluent/fluent-logger-node](http://github.com/fluent/fluent-logger-node)
+- [fluent/fluent-logger-erlang](http://github.com/fluent/fluent-logger-erlang)
+
+
+The following repositories cannot be contemplated in the donation as they are not owned by Treasure Data:
+
+- [fluent/nginx-fluentd-module](http://github.com/fluent/nginx-fluentd-module)
+- [fluent/fluent-plugin-rewrite-tag-filter](http://github.com/fluent/fluent-plugin-rewrite-tag-filter)
+- [fluent/fluent-logger-perl](http://github.com/fluent/fluent-logger-perl)
+
+*Initial Committers*:
+
+* Sadayuki "Sada" Furuhashi
+
+*Infrastructure requirements*: CI and potentially CNCF Community Cluster access
+
+We currently have several unit tests for all components of Fluentd, the tests can be found in the source code here:
+
+https://github.com/fluent/fluentd/tree/master/test
+
+With CNCF infrastructure, we aim to improve our testing with real deployments and critical loads over different connectors that we provide.
+
+*Issue tracker*: https://github.com/fluent/fluentd/issues and respective github issues for each repository.
+
+*Mailing lists:* https://groups.google.com/forum/#!forum/fluentd
+
+*Website*: http://www.fluentd.org/
+
+*Release methodology and mechanics*: The latest release is found at: https://rubygems.org/gems/fluentd
+
+* Generally the last major version and one branch of the old major version are actively supported
+* Changes are included in a minor version if they don’t contain backward incompatibility, otherwise they’re included only in a branch of the next major version
+* The current latest major version is 0.14 and the last major version is 0.12.
```

Comment:
so to be clear, fluentd doesn't strictly follow semver? (even though the version numbers suggest so) - can that be tweaked?